packages/calling-component-bindings/src/participantListSelectorUtils.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Since we have multiple utils files(another is callUtils), let's move all of these into utils folder, to make the top folder cleaner